lib/Common/ConfigFlagsList.h

```diff
@@ -1115,6 +1116,7 @@ FLAGPR           (Boolean, ES6, ES6Rest                , "Enable ES6 Rest parame
 FLAGPR           (Boolean, ES6, ES6Spread              , "Enable ES6 Spread support"                                , DEFAULT_CONFIG_ES6Spread)
 FLAGPR           (Boolean, ES6, ES6String              , "Enable ES6 String extensions"                             , DEFAULT_CONFIG_ES6String)
 FLAGPR           (Boolean, ES6, ES6StringPrototypeFixes, "Enable ES6 String.prototype fixes"                        , DEFAULT_CONFIG_ES6StringPrototypeFixes)
+FLAGPR           (Boolean, ES6, ES2019ObjectSpread     , "Enable ES2019 Object Spread"                              , DEFAULT_CONFIG_ES2019ObjectSpread)
```

Comment:
The spec proposal looks like this is ES2018?

Reply:
Ah ok. I had no idea what it was supposed to be so I just chose 2019 randomly.